src/inspect.rs

```diff
@@ -1,27 +1,65 @@
+use crate::{DockerConfig, Registry, Sources};
 use anyhow::{anyhow, Result};
 use mdcat::{ResourceAccess, TerminalCapabilities, TerminalSize};
+use policy_evaluator::policy_fetcher::oci_distribution::manifest::{OciImageManifest, OciManifest};
```

Comment:
there are some clippy warnings that are concerning, like

```
failed to resolve: could not find `oci_distribution` in `policy_fetcher`
```

Maybe we have to rerun the tests?